app/src/main/java/org/openimis/imispolicies/NotEnrolledPoliciesOverview.java

```diff
@@ -547,7 +547,11 @@ private int getControlNumber(final JSONObject order, final String SmsRequired) t
         Thread thread = new Thread() {
             public void run() {
                 HttpClient httpClient = new DefaultHttpClient();
-                HttpPost httpPost = new HttpPost(AppInformation.DomainInfo.getDomain() + "payment/GetControlNumber");
+                String domain = AppInformation.DomainInfo.getDomain();
+                if (BuildConfig.API_BASE_URL.equals("http://chf-dev.swisstph-mis.ch/rest/") || BuildConfig.API_BASE_URL.equals("http://196.192.73.26/rest/")) {
```

Comment:
we need to use the BUILD name or version instead because the address or Ip may change

Reply:
ok

Reply:
this part of code is fixed in new commit. I use "applicationId" to verify build @delcroip